commit 6a14834c7d3a3fd622bb51f9ade18750267e95d3
Author: Cole Robinson <crobinso@redhat.com>
Date:   Thu Apr 28 10:39:12 2011 -0400

    create: Bump default memory from 512 to 1024MB
    
    F15 needs more than 512 or the installer crashes, pretty sure recent windows
    require at least a GB as well. Long term plan is to track a default mem value
    per OS being installed.

diff --git a/src/virtManager/create.py b/src/virtManager/create.py
index 6bbe23d..f8da9f2 100644
--- a/src/virtManager/create.py
+++ b/src/virtManager/create.py
@@ -42,6 +42,8 @@ OS_GENERIC = "generic"
 # Number of seconds to wait for media detection
 DETECT_TIMEOUT = 20
 
+DEFAULT_MEM = 1024
+
 PAGE_NAME = 0
 PAGE_INSTALL = 1
 PAGE_MEM = 2
@@ -377,7 +379,7 @@ class vmmCreate(vmmGObjectUI):
         self.window.get_widget("install-import-entry").set_text("")
 
         # Mem / CPUs
-        self.window.get_widget("config-mem").set_value(512)
+        self.window.get_widget("config-mem").set_value(DEFAULT_MEM)
         self.window.get_widget("config-cpus").set_value(1)
 
         # Storage
